src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx

```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
+![nrexplorer2.png](./images/nrexplorer20210722.png "nrexplorer2.png")
 
 <figcaption>
   **[one.newrelic.com](http://one.newrelic.com) > Explorer**: Use the New Relic Explorer to locate and examine the entities you monitor.
 </figcaption>
 
 Toggle between the Explorer's three views and the following features:
 
-1. [List](#view-list). Browse and filter from a list of all the entities in your account. Use this landing interface to navigate, group and filter your entities.
-2. [New Relic Navigator](#view-navigator). Get a high density overview of all your entities, grouped by entity type or by tags. Use this to detect any issues and health patterns at a glance.
-3. [New Relic Lookout](#view-lookout). Spot entities that have recent performance deviations. Use this to quickly identify unusual behaviors. Other components include:
-4. [Filterbar](#filter-tag). Drill down and locate problematic entities easily with our improved search capabilities, and benefit from the `AND` and OR operators in place.
-5. See everything. All the different entity types you have access to are listed in the collapsible sidebar on the left of the screen. This sidebar is interactive and used for exploratory purposes, and it allows you to see only entities of the selected type, updating the filter from the filterbar.
-6. Create a new [workload](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster) for meaningful groupings of your monitored entities.
-7. Add more data to instrument more elements of your system and achieve full stack observability.
+1. **[List](#view-list)**. Browse and filter from a list of all the entities in your account. Use this landing interface to navigate, group and filter your entities.
+2. **[New Relic Navigator](#view-navigator)**. Get a high density overview of all your entities, grouped by entity type or by tags. Use this to detect any issues and health patterns at a glance.
+3. **[New Relic Lookout](#view-lookout)**. Spot entities that have recent performance deviations. Use this to quickly identify unusual behaviors. 
+4. **Saved views**. Save your favorite filters as a view and recover them in a click each time you return to the Explorer. 
+5. **[Filterbar](#filter-tag)**. Drill down and locate problematic entities easily with our improved search capabilities, and benefit from the `AND` and OR operators in place.
+6. **See everything**. All the different entity types you have access to are listed in the collapsible sidebar on the left of the screen. This sidebar is interactive and used for exploratory purposes, and it allows you to see only entities of the selected type, updating the filter from the filterbar.
+7. Create a new **[workload](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster)** for meaningful groupings of your monitored entities.
+8. **Add more data** to instrument more elements of your system and achieve full stack observability.
 
```

Comment:
This new section is excellent! Really helpful info, congrats!

Reply:
Thanks!